test/functional/checkObjectExists.js

```diff
@@ -169,6 +169,8 @@ function checkObjectExists(config, name, type, callback) {
     checkObjectExistsODBC(config.transportOptions, object, callback);
   } else if (config.transport === 'ssh') {
     checkObjectExistsSSH(config.transportOptions, object, callback);
+  } else { // we cannot check object existence using the rest transport
+    callback(null);
```

Comment:
```suggestion
    callback(null, true);
```

Shouldn't we pass the second parameter? Otherwise the check will get an undefined which would prevent it from running.

Reply:
Thats right, I will fix this up.